content/blog/asyncapi_codegen_scst.md

```diff
@@ -0,0 +1,142 @@
+---
+type: blog
+featured: true
+cover: /images/posts/asyncapi-codegen_pic-00.webp
+weight: 100
+title: AsyncAPI Code Generation: Microservices Using Spring Cloud Stream
```

Comment:
```suggestion
title: "AsyncAPI Code Generation: Microservices Using Spring Cloud Stream"
```

yaml doesn't like what you did here with `:` as this is wrong mapping for him. If you need colon here, then it must be in `"` to indicate directly it is a string